dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/BasicAuthTest.java

```diff
@@ -28,16 +28,17 @@
 import com.ibm.ws.webcontainer.security.test.servlets.SSLBasicAuthClient;
 import com.ibm.ws.webcontainer.security.test.servlets.SSLServletClient;
 import com.ibm.ws.webcontainer.security.test.servlets.ServletClient;
-import com.ibm.ws.webcontainer.security.test.servlets.TestConfiguration;
 
+import componenttest.annotation.CheckpointTest;
+import componenttest.annotation.SkipForRepeat;
 import componenttest.custom.junit.runner.FATRunner;
-import componenttest.custom.junit.runner.Mode;
-import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.CheckpointRule;
+import componenttest.rules.repeater.CheckpointRule.ServerMode;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.impl.LibertyServerFactory;
 
 @RunWith(FATRunner.class)
-@Mode(TestMode.FULL)
+@CheckpointTest(alwaysRun = true)
 public class BasicAuthTest extends CommonServletTestScenarios {
     private static String DEFAULT_CONFIG_FILE = "basicauth.server.orig.xml";
     private static LibertyServer myServer = LibertyServerFactory.getLibertyServer("com.ibm.ws.webcontainer.security.fat.basicauth");
```

Comment:
Should remove the assignment of the server here since we need it to happen in `serverSetup`

Reply:
Done, thanks!